koku/reporting/migrations/0156_gcp_cost_summary.py

```diff
@@ -0,0 +1,135 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("reporting", "0155_gcp_partitioned")]
+
+    operations = [
+        migrations.RunSQL(
+            """
+                DROP INDEX IF EXISTS gcp_cost_summary;
+                DROP MATERIALIZED VIEW IF EXISTS reporting_gcp_cost_summary;
+
+                DROP INDEX IF EXISTS gcp_cost_summary_account;
+                DROP MATERIALIZED VIEW IF EXISTS reporting_gcp_cost_summary_by_account;
+
+                DROP INDEX IF EXISTS gcp_cost_summary_project;
+                DROP MATERIALIZED VIEW IF EXISTS reporting_gcp_cost_summary_by_project;
+
+                DROP INDEX IF EXISTS gcp_cost_summary_region;
+                DROP MATERIALIZED VIEW IF EXISTS reporting_gcp_cost_summary_by_region;
+
+                DROP INDEX IF EXISTS gcp_cost_summary_service;
+                DROP MATERIALIZED VIEW IF EXISTS reporting_gcp_cost_summary_by_service;
+
+                CREATE MATERIALIZED VIEW reporting_gcp_cost_summary AS(
```

Comment:
Instead of manually creating these here with SQL, you can instead make a function to load them from the files you've created. See https://github.com/project-koku/koku/blob/master/koku/reporting/migrations/0146_auto_20200917_1448.py#L9-L16.

This way we don't end up with different versions of the views in migration and the file becomes the single source of truth.

Reply:
Oohhhh. That is new, I like that a lot better. I will make that change now.